src/GraphQL/Validation/ValidationContext.cs

```diff
@@ -34,11 +35,6 @@ internal void Reset()
             Inputs = null;
         }
 
-        /// <summary>
-        /// Returns the original GraphQL query string.
-        /// </summary>
-        public string OriginalQuery { get; set; }
```

Comment:
This could have been changed to a property, so as to not break existing applications, but I'm okay to remove it.
```cs
public string OriginalQuery => Document.OriginalQuery;
```

Reply:
I remove such properties if I see they go against the single source of truth.